src/components/Nav/Nav.js

```diff
@@ -1,7 +1,115 @@
 import React from 'react';
+import styled, { css } from 'styled-components';
+import { Link } from 'react-router-dom';
 
 const Nav = () => {
-  return <h1>Nav</h1>;
+  return (
+    <NavigateMain>
+      <NavMain>
+        <NavMainMenu>
+          <Link to="/">
+            <NavMainMenuImage
+              src="../images/Screenshot_20221227_112939_Samsung Notes.jpg"
+              alt="로고 이미지"
+            />
+          </Link>
+          <NavMainMenuList>
+            <StyledLink to="/subscribe">
+              <NavMainMenuListItem>구독서비스</NavMainMenuListItem>
+            </StyledLink>
+            <StyledLink to="/productlist">
+              <NavMainMenuListItem>담아마켓(스토어)</NavMainMenuListItem>
+            </StyledLink>
+            <Link to="/Search">
+              <NavMainMenuSearch
+                type="text"
+                placeholder="🔍 무엇을 찾고 계신가요?"
+              />
+            </Link>
+          </NavMainMenuList>
+        </NavMainMenu>
+        <NavSideMenuList>
+          <StyledLink to="/signin">
+            <NavSideMenuListItem>로그인</NavSideMenuListItem>
+          </StyledLink>
+          <Link to="/cart">
+            <NavSideMenuListImage
+              src="https://cdn-icons-png.flaticon.com/512/7695/7695110.png"
+              alt="장바구니 이미지"
+            />
+          </Link>
+        </NavSideMenuList>
+      </NavMain>
+    </NavigateMain>
+  );
 };
 
+const NavigateMain = styled.div`
+  position: sticky;
+  border: none;
+  border-bottom: 0.5px solid rgb(224, 224, 224);
+  padding: 18px 69px;
+  z-index: 999;
+  top: 0;
+  background-color: white;
+`;
+
+const NavMain = styled.div`
+  ${props => props.theme.mixin.flex('', 'space-around', 'center')}
```

Comment:
```suggestion
  ${({theme}) => theme.mixin.flex(null, 'space-around', 'center')}
```
굿!
적용안하게 하고싶은 값은 null을 보내면 됩니다.
구조분해할당도 할 수 있어요.
다른 코드도 수정 해 보세요~